### PR TITLE
PLT-2531 Allow customization of LDAP login fields

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -145,7 +145,9 @@
         "UsernameAttribute": "",
         "IdAttribute": "",
         "SkipCertificateVerification": false,
-        "QueryTimeout": 60
+        "QueryTimeout": 60,
+        "LoginFieldName": "",
```
Comment (crspeller) [inline config/config.json]: Shouldn't this be under branding and not LDAP?

Comment (jwilander) [inline config/config.json]: Nope, it's LDAP only so I'm pretty sure it should go here. Also there isn't really a Branding section, the branding stuff is just under Team Settings.

Comment (crspeller) [inline config/config.json]: Maybe we should have a branding section? It seems like configuring LDAP and customizing the look of the software are two different tasks. Although I might be thinking about this setting while I am configuring LDAP.  0/5

**Comment by crspeller** [comment]
+1 Ether way my comment above is handled.

File: config/config.json (hunk 1 above)
Comment (jwilander) [inline config/config.json]: The whole system console is about to be refactored anyways so I'd rather not add a Branding section right now (I'm pretty sure the Branding spec purposefully placed it in Team Settings temporarily). It's also the customizing the look of LDAP part of the software :P I don't really see it as branding

**Pull request merged**